### DISCUSSION TITLE
Document verification & Custom plugin

### DISCUSSION THREAD

**Discussion by anil4684** [Ginger-Automation/Ginger#2033] [Q&A] [answered]
Hi Team,

Could please clarify on the below .

1. How can we do the document verification using Ginger ( Like verification of text in PDF) ?
2. please share the guideline document to create Custom plugin .

Thanks,
Anil

**Comment by JinendraGhodke** [comment, marked as answer]
PDF Text verification is currently not supported.
Please find below the document on how to create a new plugin.
[Ginger Plugins KT Dev 2020.pptx](https://github.com/Ginger-Automation/Ginger/files/7656238/Ginger.Plugins.KT.Dev.2020.pptx)
